### PR TITLE
remove availabilityConfig and update storageConfig related doc

### PR DESCRIPTION
**Pull request by songleo**
we do not support `availabilityConfig` config in the observability service. so we need to remove this config.
Signed-off-by: Song Song Li <ssli@redhat.com>

### CODE REVIEW

**Comment by songleo** [comment]
@dockerymick @clyang82 Please help review it, thanks.

File: observability/observability_enable.adoc [hunk 1]
```diff
@@ -130,7 +130,6 @@ kind: MultiClusterObservability
 metadata:
   name: observability # Your customized name of MulticlusterObservability CR
 spec:
-  availabilityConfig: High # Available values are High or Basic
   enableDownSampling: true # The default value is true. This is recommended as querying long-time ranges downsampled data is efficient and useful.
```
Comment (clyang82) [inline observability/observability_enable.adoc]: s/`enableDownSampling`/`enableDownsampling`/g

Comment (clyang82) [reply observability/observability_enable.adoc]: please also update `storageConfig` section.

**Review by clyang82 — CHANGES_REQUESTED** [review]

**Comment by clyang82** [comment]
💯 Thanks
/lgtm

**Review by dockerymick — APPROVED** [review]
Thanks @songleo the changes look good! Is there a doc issue for this before we merge?

**Comment by songleo** [comment]
> Thanks @songleo the changes look good! Is there a doc issue for this before we merge?

yes, it is a doc issue, we need to update doc due to api changed. thanks

**Comment by dockerymick** [comment]
Thanks @songleo for confirming and creating the issue. This helps us to track the  reason for the updates. This PR can be merged

**Pull request merged**